source/eks.rst

```diff
@@ -0,0 +1,165 @@
+
+Open the IAM console, select Roles on the left and then click the Create Role button at the top of the page.
+From the list of AWS services, select EKS and then Next: Permissions at the bottom of the page.
+Leave the selected policies as-is, and proceed to the Review page. Enter a name for the role (e.g. eksrole) and hit the Create role button at the bottom of the page to create the IAM role. The Role ARN specified on the page is to be used later.
+
+
+Create the EKS cluster
+======================
+
+To create your cluster, you will need the following data:
+
+* name of your EKS cluster,
+* AWS region in which you wish to deploy your cluster,
+* the amount of nodes you would like tho have,
+* the ratio of `spot https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-spot-instances.html`_ and `on-demand <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-on-demand-instances.html>`_ instances to use.
+
+The most easy and visually clear way is to describe the desired cluster in YAML
+and to pass this configuration to the ``eksctl`` command. 
+
+The following example configures a EKS cluster with one `managed node group <https://docs.aws.amazon.com/eks/latest/userguide/managed-node-groups.html>`_ , including .... **ToDo**:
+
+.. code:: yaml
+
+   apiVersion: eksctl.io/v1alpha5
+   kind: ClusterConfig
+
+   metadata:
+       name: test-cluster
+       region: eu-west-2
+
+   nodeGroups:
+       - name: ng-1
+         minSize: 3
+         maxSize: 5
+         instancesDistribution:
+           maxPrice: 0.15
+           instanceTypes: ["m5.xlarge", "m5.2xlarge"] # At least two instance types should be specified
+           onDemandBaseCapacity: 0
+           onDemandPercentageAboveBaseCapacity: 50
+           spotInstancePools: 2
+         tags:
+           'iit-billing-tag': 'cloud'
+         preBootstrapCommands:
+             - "echo 'OPTIONS=\"--default-ulimit nofile=1048576:1048576\"' >> /etc/sysconfig/docker"
+             - "systemctl restart docker"
+
+.. note:: Please note, that ``preBootstrapCommands`` section is used in the
+          above example to increase the limits for the amount of opened files:
+          this is important and shouldn't be omited, taking into account the
+          default EKS soft limit of 65536 files.
+
+When the cluster configuration file is ready, you can actually create your cluster
+by the following command:
+
+.. code:: bash
+
+   $ eksctl create cluster -f ~/cluster.yaml
+
+
+**is it needed?**
+
+Now update your ``kubectl`` configuration to allow it communicating with the new cluster. You can do it using the AWS CLI ``update-kubeconfig`` command. Run it with the correct cluster-name and region:
+
+.. code:: bash
+
+   $ aws eks --region eu-west-2 update-kubeconfig --name test-cluster
+
+
+Install the Operator
+=======================
+
+1. IAM? Is it already used by kubectl? What about creating Roles and RoleBindings?
```

Comment:
I think everything should be fine after `eksctl create cluster -f ~/cluster.yaml` command